src/wormhole/session/item/rename_gotchi.rs

```diff
@@ -0,0 +1,95 @@
+use super::SessSend;
+use hcor::id::ItemId;
+use hcor::{id, item, ConfigError, Item};
+use std::fmt;
+
+#[derive(Debug)]
+pub enum Error {
+    NoSuchItemConf(ConfigError),
```

Comment:
I don't see how this error could occur outside of the context of spawning items ...